src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -161,8 +161,15 @@
       <NativeVersionLines Condition="'$(Configuration)'=='Debug'" Include="#define VER_DEBUG                   VS_FF_DEBUG" />
       <NativeVersionLines Condition="'$(Configuration)'!='Debug'" Include="#define VER_DEBUG                   0" />
       <NativeVersionLines Include="#endif" />
+      <NativeVersionFile Include="$(NativeVersionHeaderFile)" />
```

Comment:
Is NativeVersionHeaderFile set anywhere but above? If not then please prefer to split the property to a path and file there instead of depending on msbuild batching to split the directory off.